src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlLoader.java

```diff
@@ -227,6 +247,37 @@ private ImportControl getRoot() {
     }
 
     /**
+     * Utility to get a strategyOnMismatch property for "import-control" tag.
+     * @param attributes collect to get attribute from.
+     * @return the value of the attribute.
+     */
+    private static MismatchStrategy getStrategyForImportControl(final Attributes attributes) {
+        final String returnValue = attributes.getValue(STRATEGY_ON_MISMATCH_ATTRIBUTE_NAME);
+        MismatchStrategy strategyOnMismatch = MismatchStrategy.DISALLOWED;
+        if (STRATEGY_ON_MISMATCH_ALLOWED_VALUE.equals(returnValue)) {
+            strategyOnMismatch = MismatchStrategy.ALLOWED;
+        }
+        return strategyOnMismatch;
+    }
+
+    /**
+     * Utility to get a strategyOnMismatch property for "subpackage" tag.
+     * @param attributes collect to get attribute from.
+     * @return the value of the attribute.
+     */
+    private static MismatchStrategy getStrategyForSubpackage(final Attributes attributes) {
+        final String returnValue = attributes.getValue(STRATEGY_ON_MISMATCH_ATTRIBUTE_NAME);
+        MismatchStrategy strategyOnMismatch = MismatchStrategy.DELEGATE_TO_PARENT;
+        if (STRATEGY_ON_MISMATCH_ALLOWED_VALUE.equals(returnValue)) {
```

Comment:
No we cannot do this as delegateToParent is not in the same spelling as DELEGATE_TO_PARENT